### DISCUSSION TITLE
Contract date but payment is already done

### DISCUSSION THREAD

**Discussion by poudenes** [firefly-iii/firefly-iii#5682] [General]
Hi,

I have some contracts that normal get the money on 1st of month. But sometimes its a weekend, so they do it before on Friday.
What will happen with the automation of the contract?  Or do I change the payment date earlier then normal?

**Comment by JC5** [comment]
Are you talking about recurring transactions or bills (English version)?

**Comment by poudenes** [comment]
The bills... 

for example I have every first of month 600,- to pay. So bill date on 2022-01-01 (in december I started use of Firefly3)
Now the bill is payed on 2022-01-31 instead of today the 1st. 

Its now not show in the bills as payed. 

I can change the date to 1st then it is showing. But I think when I connect with nordigen it will add the payment again on 2022-01-31.

**Reply by JC5** [reply]
Yes, that will happen. There's not much i can do about it.

But Nordigen will not add it again if you edit it.

**Reply by poudenes** [reply]
Ok. Then I change the bills dates so they become in the budget, etc of februari :)

off-topic: I exported 2 files for firefly-import. Will there be any change that you will implement those settings into the app, so you have a pulldown menu to select what settings you want use to get the new information? 

First selection is none, and the rest are stored settings to make import easier.

**Reply by JC5** [reply]
Yes this is possible, see https://docs.firefly-iii.org/data-importer/help/config/

**Reply by poudenes** [reply]
Great, I have to see how to do this. Because my docker is part as a "Add-on" inside Home Assistant ;)
